src/Component/App.test.js

```diff
@@ -0,0 +1,30 @@
+import App, { Item, Thumbnail, Title } from './App'
+import React from 'react'
+import TestRenderer from 'react-test-renderer'
+
+
+let inspector;
+
+
+beforeEach(() => {
+  inspector = TestRenderer.create(<App />)
+})
+
+
+it('display the title', () => {
+  expect(inspector.root.findByType(Title).props.children).toEqual('Welcome')
+});
+
+
+it('display some quick access link', () => {
+  const items = inspector.root.findAllByType(Thumbnail)
+
+  expect(items.length).toBe(4)
+});
+
+
```

Comment:
Why to have to charet return between each test ?

Reply:
SInce a function may contains "one chare return" inside there body, i find it more readable to separate them (i mean the functions) by two charet returns. Many people in the FP world are using this technic. It allows to put some charet return in functions as you want and separate this functions by two charret returns is a pleasure for the eye ^^